### ISSUE TITLE
Need `ContainerRepository` option

### ISSUE THREAD

**Issue by justinyoo** [prom3theu5/aspirational-manifests#112]
### 🔥 Bug Description

If we use Docker Hub as the container registry, we might have to specify the `ContainerRepository` node on our `.csproj` file. [Reference](https://github.com/dotnet/sdk-container-builds/blob/main/docs/RegistryAuthentication.md#notes-for-specific-registries)

Therefore, for this particular purpose, the `ContainerRepository` option should be provided as a fall-back option.

### 🧯 Possible Solution

Currently, we use names defined in the `.AppHost` project (eg. `apiservice` and `webfrontend`), which becomes the container image names. But for the Docker Hub case, the image names should be prefixed with the username like `<docker_username>/apiservice` and `<docker_username>/webfrontend` respectively. These values should be explicitly set in their respective `.csproj` files, to publsh the container image to Docker Hub.

Is there a chance to add a new option like `--container-repository-prefix`, so that the fall-back option takes care of this value?

Here's the suggested CLI command like:

```bash
aspirate init --container-registry registry.hub.docker.com --container-repository-prefix contoso ...
```

Then, the generated `aspirate.json` might look like:

```json
{
  "TemplatePath": null,
  "ContainerSettings": {
    "Registry": "registry.hub.docker.com",
    "RepositoryPrefix": "contoso"
  }
}
```

Then, if we run the `aspirate generate` command, it will run the `dotnet publish` command like:

```bash
dotnet publish "/path/to/Contoso.AppHost/../Contoso.ApiService/Contoso.ApiService.csproj" \
    -p:PublishProfile="DefaultContainer" \
    -p:PublishSingleFile="true" \
    -p:PublishTrimmed="false" \
    --self-contained "true" \
    -r "linux-x64" \
    -p:ContainerRegistry="registry.hub.docker.com" \
    -p:ContainerRepository="contoso/apiservice" \
    -p:ContainerImageTag="latest"
```

**Comment by prom3theu5** [comment]
Sure thing

**Comment by prom3theu5** [comment]
released in `v0.1.36-preview`

**Comment by justinyoo** [comment]
Thanks @prom3theu5 for updating it! Let me try it.
